scripts/process.py

```diff
@@ -114,6 +116,19 @@ def fix_canada_recovered_data(rows):
             row['Long'] = row.get('Long', '-106.3468')
         yield {**expected, **row}
 
+        
+## Define parameters for cleaning
```

Comment:
This is reasonable, but again we do not know if this applies only for lines 122 to 124 or also for lines 126 and 127 or even until the end of the file: after all, the comment on line 21 would theoretically apply at least until here. Comments should also generally be left without a blank line below so we have visual feedback for block of code/comments that belong together.